apps/server/src/apps/helpers/app-logger-middleware.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response, NextFunction } from 'express';
+import { Logger } from '@src/core/logger';
+import { RequestLoggable } from '@src/apps/helpers/request-loggable';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+
+export const createAppLoggerMiddleware = (
+	logger: Logger
+): ((request: Request, response: Response, next: NextFunction) => void) => {
+	logger.setContext('AppLoggerMiddleware');
```

Comment:
Hier wird der context immer wieder je request neu zugewiesen. Nicht dramatisch, aber auch nicht sinnvoll.

Reply:
nope, just once, line 12 is the function that gets returned, thats the code that happens every request